components/search/SearchSuggestion.vue

```diff
@@ -573,6 +586,16 @@ export default class SearchSuggestion extends mixins(PrefixMixin) {
     }
   }
 
+  getFloorPrice(nfts) {
+    if (!nfts) {
+      return 0
+    }
+    return nfts
+      .map((x) => Number(x.price))
+      .sort()
+      .at(0)
```

Comment:
```suggestion
return Math.min(...nft.map(nft => Number(nft.price)))
```

not efficient to use sort when you just want min